README.md

```diff
@@ -831,8 +833,17 @@ message AgentAddonStatus {
 Addon name. MUST be always set and MUST match the key in the addons field of
 AgentAddonStatuses message.
 
-<h4 id="agent_has_hash">agent_has_hash</h4>
+#### agent_has_version
 
+The version of the addon that the Agent has.
```

Comment:
IIRC we discussed possibility for Agent having several addon versions installed locally (e.g. when the new one fails to be able to do a rollback). Should we add a clarification that when the agent has multiple addon versions, the most recent version should be reported (or perhaps the one that is enabled?)

Reply:
Can we leave the decision to the Agent and leave unspecified in the protocol?

Reply:
Yeah, I think that should be OK for now and if we ever need to change that, we could revisit